prisma/schema.prisma

```diff
@@ -205,11 +214,10 @@ enum RequestType {
   DELETE
 }
 
-// enum PublicCloudRequestType {
-//   CREATE
-//   EDIT
-//   DELETE
-// }
+enum PublicCloudRequestType {
+  CREATE
+  EDIT
+}
```

Comment:
Public cloud has the same Request types as Private - delete, edit and create

Reply:
Thank you, I will update this